src/dhtproto/client/DhtClient.d

```diff
@@ -711,30 +783,65 @@ public class DhtClient : IClient
 
     /***************************************************************************
 
+        Constructor with support for the neo and legacy protocols. Automatically
+        calls addNodes() with the node definition files specified in the legacy
+        and neo Config instances.
+
+        Params:
+            epoll = select dispatcher to use
+            config = swarm.client.model.IClient.Config instance. (The Config
+                class is designed to be read from an application's config.ini
+                file via ocean.util.config.ClassFiller.)
+            neo_config = swarm.neo.client.mixins.ClientCore.Config instance.
+                (The Config class is designed to be read from an application's
+                config.ini file via ocean.util.config.ClassFiller.)
+            conn_notifier = delegate which is called when a connection attempt
+                succeeds or fails (including when a connection is
+                re-established). Of type:
+                void delegate ( IPAddress node_address, Exception e )
+            fiber_stack_size = size (in bytes) of stack of individual connection
+                fibers
+
+    ***************************************************************************/
+
+    public this ( EpollSelectDispatcher epoll, IClient.Config config,
+        Neo.Config neo_config, Neo.DhtConnectionNotifier conn_notifier,
+        size_t fiber_stack_size = IClient.default_fiber_stack_size )
+    {
+        with ( config )
+        {
+            this(epoll, connection_limit(), queue_size(), fiber_stack_size);
+
+            this.addNodes(nodes_file);
+        }
+
+        this.neoInit(neo_config, conn_notifier);
+    }
+
+
+    /***************************************************************************
+
         Constructor with support for the neo and legacy protocols. This
         constructor that accepts all arguments manually (i.e. not read from
         config files) is mostly of use in tests.
 
         Params:
-            epoll = select dispatcher to use
```

Comment:
Rest of the changes in this file should go in the doc tidy-up commit.

Reply:
Ah yeah, I see it's got a bit mixed up.